org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Replay/ImportReplay.java

```diff
@@ -1663,7 +1656,24 @@ private final void importRep(final String fName, final InputStream is,
       }
     }
   }
-
+  
+  private void handleException(final Exception e, final int lineCount,
+      final String line, final String fName, final String message)
+  {
+    // produce the error message
+    SwingUtilities.invokeLater(new Runnable()
```

Comment:
@ayeshare - I'm nervous about this, since we use ImportReplay in Full Debrief.  I don't thing SWT apps like calling SwingUtilities.